grub2-efi/install

```diff
@@ -11,17 +11,23 @@ case "$target" in
   i?86 ) target=i386 ;;
   x86_64 | amd64 ) target=x86_64 ;;
   aarch64 ) target=arm64 ;;
-  armv7l ) target=arm ;;
+  arm* ) target=arm ;;
 esac
 
 target="$target-efi"
 
 echo "target = $target"
 
+# Systems without efivars should just fall back to install grub as removable
```

Comment:
EFI has 2 boot paths. The default is that there is a target file listed in the boot list. The boot list is stored in NVRAM.

If no entry in the boot list was bootable (or a removable media is in the boot list), EFI falls back to removable media booting which loads a default file from /efi/boot/boot<arch>.efi.

With my U-Boot EFI patches we can run EFI binaries and expose most interfaces, but we do not have NVRAM because we would have to store that on the same flash that Linux may be running on, creating device ownership conflicts. So on those systems we instead have to rely on the removable boot case.

Please keep in mind that this shouldn't kick in on server style ARM hardware, as that boots using TianoCore and usually has dedicated flash/eeprom/something to store the efi environment in. This patch is only required for small devboard style devices (think Raspberry Pi).